File: Java/UberVan.java
import java.util.ArrayList;
import java.util.Map;

public class UberVan extends Car {
    Map<String, Map<String, Integer>> typeCarAccepted;
    ArrayList<String> seatsMaterial;
    //private Integer passengers;

    public UberVan (String license, Account driver/*, Map<String, Map<String, Integer>> typeCarAccepted, ArrayList<String> seatsMaterial*/) {
        super(license, driver);
        //this.typeCarAccepted = typeCarAccepted;
        //this.seatsMaterial = seatsMaterial;
    }

    @Override
    public void setPassenger(Integer passenger) {
        if(passenger == 6) {
            this.passengers = passenger;
        } else {
            System.out.println("Debes asignar 6 pasajeros");
        }        
    }
}

File: PHP/uberVan.php
<?php
require_once('car.php');
class UberVan extends Car {
    //public $typeCarAccepted;
    //public $seatMaterial;

    public function __construct($license, $driver/*, $typeCarAccepted, $seatMaterial*/) {
        parent::__construct($license, $driver);
        //$this->typerCarAccepted = $typeCarAccepted;
        //$this->seatMaterial = $seatMaterial;
    }

    public function setPassenger($passenger) {
        if ($passenger == 6) {
            $this->passenger = $passenger;
        }
        else {
            echo "Necesitas asignar 6 pasajeros";
        }
    }
}